File: java_courses/factorialExample.java
//    System.out.println(fact);
//    System.out.println(fact);

class FactorialClass
{
    public static void main(String[] args) {
        int fact=1;
        int number=5;
        for (int j = 1; j <= number; j++) {
            fact=fact*j;
        }
        System.out.println("Factorial of "+number+"is: "+fact);

    }
    
}

File: c++_courses/factorialExample.cpp
#include <iostream>




int main(int argc, char const *argv[])
{
	int number=5;
	int fact=1;
	
	for (int i = 1; i <=number; ++i)
	{
		fact=fact*i;
	}
	std::cout<<"The factorial number of "<<number<<" is: "<<fact<<std::endl;
	return 0;
}